marklogic-data-hub/src/test/java/com/marklogic/bootstrap/TestAppInstaller.java

```diff
@@ -203,6 +204,9 @@ public void run() {
 
             HubTestBase.applyDatabasePropertiesForTests(hubConfig);
 
+            Group group = new Group(api, "Default");
```

Comment:
This would benefit from a comment to explain why this is being enabled.

Reply:
I will add a comment